docs/index.md

```diff
@@ -87,12 +87,40 @@ Installing the latest prometheus plugin will create a config file (`~/.steampipe
 connection "prometheus" {
   plugin = "prometheus"
   address = "http://localhost:9090"
+  metrics = ["http_requests_.*", ".*error.*"]
 }
 ```
 
 - `address` - HTTP address of your prometheus server
+- `metrics` - List of metric expressions to be matched against while creating dynamic tables
 
 ## Get involved
 
 - Open source: https://github.com/turbot/steampipe-plugin-prometheus
 - Community: [Slack Channel](https://steampipe.io/community/join)
+
+## Metric Filtering
+
+By default, the prometheus plugin will pull in all available metrics from the target address which can cause timeout/performance issues. The `metrics` configuration parameter will come in handy in such a situation:
```

Comment:
Is this the current behaviour, i.e., creating dynamic tables for all metrics if no metrics are specified in the config file, or is the doc incorrect and we do not attempt to create any dynamic tables?

If we do create dynamic tables for all metrics by default, can you please update the code to only attempt to create dynamic tables if `metrics` is specified?